utils/setStatus_test.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-env node, mocha */
+'use strict';
+
+const assert = require('chai').assert
+    , setStatus = require('./setStatus')
+    , fakeResFilled = {
+        statusCode: 404
+        , statusMessage: 'Not Found'
+        , setStatus: setStatus()
+    };
+
+let fakeRes;
+
+describe('setStatus function of response object', () => {
+    beforeEach(() => {
+        fakeRes = {};
+        fakeRes.setStatus = setStatus();
```

Comment:
this can be merged with the object creation:
```
{
  setStatus: setStatus
}
```

This one will also need to change with the revision to setStatus